funnel/models/commentvote.py

```diff
@@ -13,13 +13,18 @@
 
 # --- Constants ---------------------------------------------------------------
 
+
 class COMMENT_STATE(LabeledEnum):  # NOQA: N801
     # If you add any new state, you need to add a migration to modify the check constraint
     PUBLIC = (0, 'public', __("Public"))
     SCREENED = (1, 'screened', __("Screened"))
     HIDDEN = (2, 'hidden', __("Hidden"))
     SPAM = (3, 'spam', __("Spam"))
-    DELETED = (4, 'deleted', __("Deleted"))  # For when there are children to be preserved
+    DELETED = (
+        4,
+        'deleted',
+        __("Deleted"),
+    )  # For when there are children to be preserved
```

Comment:
If we move the comment to above or below this line, it'll restore the items like the ones above.